docs/sharing.md

```diff
@@ -17,18 +17,21 @@ In many ways, the sharing by web link works similarly to products like Google Dr
 
 webKnossos sharing is tightly integrated with user permissions and access rights. See the sections [on dataset management](./datasets.md#general) and [user administration](./users.md) for more info.
 
-## Sharing 
-As mentioned earlier, any webKnossos resource can be shared as a normal web URL. The easiest way to obtain this sharing link is by clicking the `Share` button in the toolbar at the top of the screen next to the position/coordinate section (*Share icon*). 
+## Dataset Sharing
 
-For more control, you can bring up the detailed sharing dialog. The sharing dialog is accessible from the menu next to the Save button at the top of the screen under `Share`. The sharing menu includes fine-grained options for internal and outside sharing, explained in more detail below:
+ Dataset sharing allows outside users to view your datasets and segmentation layers within webKnossos.
+ Shared resources can be accessed through a direct URL or can be featured. 
+ [Please contact us](mailto:hello@webknossos.org) to feature your dataset on https://webknossos.org.
 
-1. `Private`: Only you and your team manager have access to the annotation.
-2. `Internal`: All members of your organization have access to the annotation. 
-3. `Public`: Everybody, regardless their login status, can access this annotation.
-4. (`Team Sharing`: Share this annotation with other organization members on the dashboard in the `Shared Annotations` tab)
+ The easiest way to obtain this sharing link is by clicking the `Share` button in the toolbar at the top of the screen next to the position/coordinate section (*Share icon*). 
+ In any case, the sharing link also encodes additional information, such as your current camera position/rotation, zoom level, any layers that are turned on/off, the selected node, etc ([Details below](#sharing_link_format). In other words, a sharing link typically captures your current webKnossos state so that any recipient can take off from the same situation.
```

Comment:
```suggestion
The sharing link also encodes additional information, such as your current camera position/rotation, zoom level, any layers that are turned on/off, the selected node, etc ([Details below](#sharing_link_format). In other words, a sharing link typically captures your current webKnossos state so that any recipient can take off from the same situation.
```